sw/host/opentitanlib/src/transport/hyperdebug/c2d2.rs

```diff
@@ -0,0 +1,65 @@
+use anyhow::{bail, Result};
+use std::rc::Rc;
+
+use crate::io::gpio::{GpioPin, PinMode, PullMode};
+use crate::transport::hyperdebug::{Error, Flavor, Inner, StandardFlavor, VID_GOOGLE};
+
+pub const PID_C2D2: u16 = 0x5041;
```

Comment:
This is only used inside of C2d2. Maybe move it in as an associated constant?

Reply:
Done, though it is not terribly elegant (or did I miss something.)  I almost feel that adding the constant directly in the body of the method is explanation enough.
```
fn default_usb_pid() -> u16 { 0x5041 }
```

Reply:
That is what I did, and do not find terribly elegant, since I had to introduce an impl block for the sole purpose of containing the declaration of the const.  (My compiler does not allow me to put a const inside the impl section dealing with the implementation of a trait, only in the "plain" impl section, which I did not need or have until now.)

```
impl Flavor for C2d2Flavor {
  const PID_C2D2: u16 = 0x5041;
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not a member of trait `Flavor`
```

Do you prefer the const as the single declaration in an impl, or the hex number directly in the method implementation?